genesyscloud/outbound_messagingcampaign/resource_genesyscloud_outbound_messagingcampaign_schema.go

```diff
@@ -207,53 +242,77 @@ func ResourceOutboundMessagingCampaign() *schema.Resource {
 				Description: `The order in which to sort contacts for dialing, based on up to four columns.`,
 				Optional:    true,
 				Type:        schema.TypeList,
-				Elem:        OutboundmessagingcampaigncontactsortResource,
+				Elem:        ContactSortResource,
 			},
 			`messages_per_minute`: {
 				Description: `How many messages this messaging campaign will send per minute.`,
 				Required:    true,
 				Type:        schema.TypeInt,
 			},
+			`rule_set_ids`: {
+				Description: `Rule Sets to be applied while this campaign is sending messages`,
+				Optional:    true,
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			}, // /api/v2/outbound/digitalrulesets
 			`contact_list_filter_ids`: {
 				Description: `The contact list filter to check before sending a message for this messaging campaign.`,
 				Optional:    true,
 				Type:        schema.TypeList,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			`errors`: {
+				Description: `A list of current error conditions associated with this messaging campaign.`,
+				Optional:    true,
+				Type:        schema.TypeList,
+				Elem:        restErrorDetailResource,
+			},
+			`dynamic_contact_queueing_settings`: {
+				Description: `Indicates (when true) that the campaign supports dynamic queueing of the contact list at the time of a request for contacts.`,
+				Optional:    true,
+				Type:        schema.TypeList,
```

Comment:
`dynamic_contact_queueing_settings` was Computed before. Was this change made on purpose?